js/script.js

```diff
@@ -1,29 +1,65 @@
-function calcSum() { 
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c1 = a + b;
-    document.getElementById('result').innerHTML = 'result ' + c1;   
- } 
- 
- function calcSub() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c2 = a - b;
-    document.getElementById('result').innerHTML = 'result ' + c2;   
- }
+let globalNumbers = document.querySelectorAll('.action-bar__numbers'),
+    operators = document.querySelectorAll('.action-bar__operators'),
+    cleanEntry = document.getElementById('ce'),
+    output = document.getElementById('output'),
+    xNumber = null,
+    yNumber = '',
+    operator = '',
+    operatorClicked = false;
+    output.value = '0';
 
- function calcMul() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c3 = a * b;
-    document.getElementById('result').innerHTML = 'result ' + c3;   
- }
+for (let i = 0; i < globalNumbers.length; i++) {
+    globalNumbers[i].addEventListener('click', event => 
+    numberOnClick(event.target.defaultValue)
+    );
+}
 
- function calcDiv() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c4 = a / b;
-    document.getElementById('result').innerHTML = 'result ' + c4;   
- }
+for (let i = 0; i < operators.length; i++) {
+    operators[i].addEventListener('click', event =>
+    operation(event.target.defaultValue)
+    );
+}
 
- 
+cleanEntry.addEventListener('click', event =>
+clean(event.target.defaultValue)
+);
+
+function numberOnClick(num) { 
+    
+    if (operatorClicked === true) {
```

Comment:
It can be simplified. The following expressions:

- `if (operatorClicked === true)`
- `if (operatorClicked)`

are the same.